src/js/components/Box/index.d.ts

```diff
@@ -37,6 +37,6 @@ export interface BoxProps {
   wrap?: boolean | "reverse";
 }
 
-declare const Box: React.ComponentClass<BoxProps & JSX.IntrinsicElements['div']>;
+export type BoxProps = React.ComponentClass<BoxProps & JSX.IntrinsicElements['div']>;
 
 export { Box };
```

Comment:
So we have to change here as well?

Reply:
Yea, I'm wondering how it worked without this change.

Reply:
Since both were named BoxProps it wasnt passing line 40 to tableCell it was passing the interface that was what was working so when I changed BoxProps to BoxTypes it was giving me the errors again because I was exporting tableCell as a React Function component.

Reply:
I think it might be a good time to add a tslint for our .ts files. maybe see if it will catch this or anything similar.

Reply:
Since it is only type-definitions and not ts code, it might be more complicated to do so https://stackoverflow.com/questions/49814305/how-to-compile-d-ts-typescript